tools/metadatastore-upgrade/main.go

```diff
@@ -0,0 +1,152 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"flag"
+	"fmt"
+	"log"
+	"path/filepath"
+	"os"
+	"time"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+	v1 "k8s.io/client-go/kubernetes/typed/apps/v1"
+	"k8s.io/client-go/tools/clientcmd"
+	"k8s.io/client-go/util/homedir"
+	"k8s.io/client-go/util/retry"
+
+
+	_ "k8s.io/client-go/plugin/pkg/client/auth/gcp"
+)
+
+var (
+	kubeconfigPath = flag.String("kubeconfig", "", "Absolute path to kubeconfig file.")
+	imageTag = flag.String("image_tag", "", "Image tag to use for upgrade")
+	deploymentNamespace = flag.String("namespace", "kubeflow", "Namespace where MetadataStore is deployed.")
+)
+
+const (
+	ML_METADATA_DEPLOYMENT = "metadata-deployment"
```

Comment:
How about changing this to `metadata-grpc-deployment` to be aligned with kubeflow/pipelines#2889  ?

Reply:
Agreed. Will change this and the documentation once #2889 is merged

Reply:
Ping for this, since the PR got merged :smile:

Reply:
Done.